go/exorelay/exorelay_test.go

```diff
@@ -17,16 +17,18 @@ import (
 func FeatureContext(s *godog.Suite) {
 	var exoInstance *ExoRelay
 	exocom := exocomMock.New()
-	go exocom.Listen(4100)
+	go func() {
+		err := exocom.Listen(4100)
+		if err != nil {
+			fmt.Printf("ERROR: cannot start exocomMock: %s", err)
```

Comment:
This is my quick and dirty attempt at checking the return value of `exocom.Listen`, which the linter correctly calls us out for. Strange that the type checker didn't complain about this.